README.md

```diff
@@ -83,37 +83,37 @@ Make sure to add relevant API keys inside `.env` file (by copying provided `.env
 To run all tests:
 
 ```
-npm run test
+yarn test
 ```
 
 To run unit tests only:
 
 ```
-npm run test:unit
+yarn test:unit
 ```
 
 To run integration tests only:
 
 ```
-npm run test:integration
+yarn test:integration
 ```
 
 To run gas reporting tests (via [hardhat-gas-reporter](https://hardhat.org/plugins/hardhat-gas-reporter.html)) only:
 
 ```
-npm run test:gas
+yarn test:gas
 ```
 
 To run tests with coverage (via [solidity-coverage](https://github.com/sc-forks/solidity-coverage)) reporting:
 
 ```
-npm run test:coverage
+yarn test:coverage
 ```
 
 ## Deployment
 
 Make sure that an ETH node is accessible and that the network being deployed to is supported by the `hardhat.config.ts` configuration.
 
 ```
-npm run deploy
+yarn deploy
```

Comment:
IIUC this just calls `npx hardhat deploy` 

Since by default `hardhat-deploy` executes all files in the `deploy` folder recursively this won't work as we discussed on discord. 

I see two solutions: 

1.  in `package.json` change the command to `npx hardhat deploy --tags Poll,Contracts`
2. Move the two config files in the `deploy` folder to a separate folder

either is fine by me